### PR TITLE
Refacored entities list views and SearchableViewMixin, both are using table widget and rows

### PR DESCRIPTION
**Pull request by abalakh**
Summary of changes:
* Entity list views are now correctly using Table widget instead of search+relative locators
* SearchableViewMixin now relies on `table` class attr and returns `table.read()` as search result
* Updated all entities helpers to use table column widgets instead of direct links/buttons
* Added few content view helpers and navigate steps for upcoming test coverage
* Added few notifications asserts and dismisses where they could overlap widgets and cause failures
* Fixed SatTab's `is_displayed` property, as for some reason it was still 'visible' to selenium when actually hidden on UI
* Made LCESelector widget use relative xpaths when selecting some lce, so now it will select 'Library' from specific LCESelector, not the first one found on page
* Updated SatTable and SatSecondaryTab to support newly found table/tab with different style
* Other small fixes

### CODE REVIEW

File: airgun/entities/contentview.py [hunk 1]
```diff
@@ -102,8 +134,46 @@ def prerequisite(self, *args, **kwargs):
...
     def step(self, *args, **kwargs):
-        self.parent.search(kwargs.get('entity_name'))
-        self.parent.edit.click()
+        entity_name = kwargs.get('entity_name')
+        self.parent.search(entity_name)
+        self.parent.table.row(name=entity_name)['Name'].widget.click()
+
+
+@navigator.register(ContentViewEntity, 'AddPuppetModule')
+class AddNewPuppetModule(NavigateStep):
```
Comment (oshtaier) [inline airgun/entities/contentview.py]: remind me to discuss necessity of that class tomorrow

File: airgun/views/common.py [hunk 2]
```diff
@@ -81,7 +84,8 @@ class listremove(SatSecondaryTab):
             TAB_NAME = 'List/Remove'
     """
     ROOT = ParametrizedLocator(
-        './/nav[@class="ng-scope"]/following-sibling::div')
+        './/nav[@class="ng-scope" or not(@class) and not(@data-block)]'
```
Comment (oshtaier) [inline airgun/views/common.py]: anything simpler?

Comment (abalakh) [reply airgun/views/common.py]: i found secondary tab which has no attrs at all - just `<nav>`

File: airgun/entities/contentview.py (hunk 1 above)
Comment (oshtaier) [reply airgun/entities/contentview.py]: skip on this

File: airgun/views/common.py (hunk 2 above)
Comment (oshtaier) [reply airgun/views/common.py]: ok, spend 20 mins more on that per our discussion

**Comment by abalakh** [comment]
@oshtaier addressed.
Test results:
```python
pytest -v tests/foreman/ui_airgun/test_contentview.py -k test_positive_create_composite
============================= test session starts ==============================
platform darwin -- Python 3.6.4, pytest-3.4.0, py-1.5.3, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: wait-for-1.0.9, xdist-1.22.0, services-1.2.1, mock-1.6.3, forked-0.2, env-0.6.2
collected 10 items
2018-05-18 16:57:22 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/ui_airgun/test_contentview.py::test_positive_create_composite PASSED [100%]

============================== 9 tests deselected ==============================
=================== 1 passed, 9 deselected in 319.15 seconds ===================
```

**Pull request merged**